tests/integrations/client/http_client_test.go

```diff
@@ -397,6 +397,30 @@ func (suite *httpClientTestSuite) TestAccelerateSchedule() {
 	re.Len(suspectRegions, 2)
 }
 
+func (suite *httpClientTestSuite) TestConfig() {
+	re := suite.Require()
+	config, err := suite.client.GetConfig(suite.ctx)
+	re.NoError(err)
+	re.Equal(float64(4), config["schedule"].(map[string]interface{})["leader-schedule-limit"])
+
+	newConfig := map[string]interface{}{
+		"schedule.leader-schedule-limit": float64(8),
+	}
+	err = suite.client.SetConfig(suite.ctx, newConfig)
+	re.NoError(err)
+
+	config, err = suite.client.GetConfig(suite.ctx)
+	re.NoError(err)
+	re.Equal(float64(8), config["schedule"].(map[string]interface{})["leader-schedule-limit"])
+
+	// Test the config with TTL.
+	newConfig = map[string]interface{}{
+		"schedule.leader-schedule-limit": float64(16),
+	}
+	err = suite.client.SetConfig(suite.ctx, newConfig, 1)
```

Comment:
Better validate whether the TTL takes effect.

Reply:
unit test shows `GetConfig` will not see TTL config, can you help me verify if it's expected?

Reply:
After checking the code, it seems the only way to read the TTL config from outside is to access etcd directly.

Reply:
https://github.com/tikv/pd/pull/7699/commits/6aa2ce91672480a034f445b7e1a2a5352156ca3e